src/lib/wizards/functions/steps/templateConfiguration.svelte

```diff
@@ -4,14 +4,23 @@
     import { FormList, InputSelect, InputText } from '$lib/elements/forms';
     import { WizardStep } from '$lib/layout';
     import { runtimesList } from '$lib/stores/runtimes';
+    import { specificationsList } from '$lib/stores/specifications';
+    import { BillingPlan } from '$lib/constants';
+    import SpecificationsTooltip from '../components/specificationsTooltip.svelte';
     import { template, templateConfig } from '../store';
+    import { organization } from '$lib/stores/organization';
+    import { isCloud } from '$lib/system';
 
     let showCustomId = false;
 
     async function beforeSubmit() {
         if (!$templateConfig.runtime) {
             throw new Error('Please select a runtime.');
         }
+
+        if (!$templateConfig.specification) {
+            throw new Error('Please select a specification.');
+        }
```

Comment:
Can you try this flow?
I think template without specification is okay - it can take default value

Reply:
We automatically select the base specification on load, see https://github.com/appwrite/console/pull/1301/files/cedc1af747e9c1dfb3a2d4190f4a10e6e6ff99f1#diff-b1d4bcc01b5994925abf0fc4b457c6a8e50804765145f043d9c3ffe8a4b062d6R48-R51